README.md

```diff
@@ -45,20 +45,20 @@ If you have not been specifically asked, you may pick either `Implementation Pat
 
 ### Basic Requirements
 
-  - Use Node.js `LTS` and any framework of your choice.
-  - Use any persistence store. NoSQL DB is preferred.
-  - Write concise and clear commit messages.
-  - Write clear **documentation** on how it has been designed and how to run the code.
+- Use Node.js `LTS` and any framework of your choice.
+- Use any persistence store. NoSQL DB is preferred.
+- Write concise and clear commit messages.
+- Write clear **documentation** on how it has been designed and how to run the code.
```

Comment:
I am using VSCode with an automatic lint on write.  I should probably add an eslint config to match your standards.

Reply:
> How do you prefer to handle linting / auto-prettifying standards across multiple repositories?

For the most part, I care that standards exist, and that code cannot be put into master unless it has passed an auto linter.  I have been on a team that attempted to use a npm package to import into various repos for this purpose.  In our case, it wasn't seamless, but I think this is a fine approach.  On the other hand I don't think its a problem to have each repo's linting managed by the source maintainers -- especially if repos tend to vary in stack.

>For example, have you worked with any automation / CI hooks that check for linting or automatically prettify code based on config?

I have used Travis CI to run linters, tests before allowing to merge to master.  For a side project earlier this year I used CircleCI to do something similar.